### ISSUE TITLE
Exit code when translations missing

### ISSUE THREAD

**Issue by stevenkuster** [pixari/vue-i18n-extract#56]
Hi, I have a question. I would like to use vue-i18n-extract to fail my Semaphore build when translations are missing. However, the plugin returns an exit code 0 and the build passes even when it has missing translation keys. Is this a feature that can be added? Or am I just missing something?

Thanks in advance for the reply!

**Comment by pixari** [comment]
Hi! 
Actually I had the same issue the last week and I used the library inside a git hook. 

I think we could improve the library adding this feature. 
I don't know if it should be something that belongs to the library or an external project.

What do you think about it, @Spittal?

**Comment by Spittal** [comment]
Will take a look!

**Comment by stevenkuster** [comment]
Thanks, that would be awesome!

**Comment by mlocati** [comment]
I do agree, an exit code would be a great help in CI.

I managed to solve this by inspecting the contents of the generated report:

```bash
npm run vue-i18n-extract -- report -v './src/**/*.vue' -l './src/i18n/*.ts' -o i18n-report.json

if test ! grep -q '{"missingKeys":\[\],"unusedKeys":\[\]}' i18n-report.json; then
    // some keys missing and/or some unused keys present
fi
```

For example, a TravisCI job could be something like this:

```sh
#!/bin/sh

set -o errexit

npm run vue-i18n-extract -- report -v './src/**/*.vue' -l './src/i18n/*.ts' -o i18n-report.json

grep -q '{"missingKeys":\[\],"unusedKeys":\[\]}' i18n-report.json
```

The script will exit with 0 (success) of the report.json file contains `{"missingKeys":[],"unusedKeys":[]}`, with non-0 (failure) otherwise.

**Comment by erickwilder** [comment]
The status code is the most sane way to solve this because sniffing the output makes these solutions really fragile as this project evolves. If, for some reason the output changes, then all these scripts have to be updated.

**Comment by pixari** [comment]
I agree.
Exit codes still have to be implemented in the library.
Unfortunately I didn't have time to work on it and this proposal is still "on hold".

Feel free to propose a technical implementation, I'll work on it in the next weeks.

**Comment by wesleymostien** [comment]
Really looking forward for this ;-)

Then I can use this in my Git Hooks before a commit.

**Comment by pixari** [comment]
I am working on it, @wesleymostien :)

**Comment by pixari** [comment]
Hello @stevenkuster!

The feature has been implemented: https://github.com/pixari/vue-i18n-extract/pull/108

**Comment by stevenkuster** [comment]
> Hello @stevenkuster!
> 
> The feature has been implemented: #108

Awesome! Thanks for this
